applications/luci-app-nlbwmon/root/etc/uci-defaults/40_luci-nlbwmon

```diff
@@ -1,11 +1,4 @@
 #!/bin/sh
 
-uci -q batch <<-EOF >/dev/null
```

Comment:
https://github.com/openwrt/packages/blob/5da8ae16b23293250ac6c28bd727101b9c539500/net/nlbwmon/files/nlbwmon.init#L102
nlbwmon doesn't contain a trigger for config `nlbwmon`, only for `dhcp` and `system`. Should I keep this?

Reply:
good question.
Maybe we should add the vpnbypass file to the trigger service?

Reply:
@jow-

Reply:
Please add a trigger for the nlbwmon config